src/server/templates/base.html

```diff
@@ -50,12 +50,21 @@
     <header>
       <div class="container">
         <div class="row d-flex align-items-center">
-          <h1 class="col-3 flex-grow-1">Epidemic Forecasting</h1>
+          <h1 class="col-3 flex-grow-1"><a href="/">Epidemic Forecasting</a></h1>
           <nav class="col nav nav-masthead justify-content-end">
-            <a class="nav-link active" href="/">Home</a>
-            <a class="nav-link" href="/model">Model</a>
-            <a class="nav-link" href="/request-calculation">Request model</a>
-            <a class="nav-link" href="/request-event-evaluation">Events Prediction</a>
+            {% set navigation_bar = [ ('/', 'index', 'Public information'),
+            ('/model/', 'models', 'Models'), ('/request-calculation/',
```

Comment:
What about putting this somewhere higher up, so that it does not need to be indented as much? Or even a separate file.

Reply:
I would use `url_for` rather than the direct path

Reply:
Done

Reply:
Moved up.

Given that we haven't changed these that much, Would go with the absolute paths for now. Is that OK with you or do you want it now?

Reply:
I find it more robust, but we can set it as future issue

Reply:
https://github.com/epidemics/covid/issues/76